README.md

```diff
@@ -59,7 +65,7 @@ Fill in the attributes as applicable to your board:
 
 ### Attributes
 
-The following attributes are available for `viam:raspberry-pi:rpi` board:
+The following attributes are available for `viam:raspberry-pi:<raspberry-pi-model>` board:
```

Comment:
can't comment on the exact line but should the pull up/down attributes be added to config attributes above?

Reply:
pulls are described in the pins section, though I agree that the order/structure of the readme is kinda hard to follow

Reply:
can be explicit in saying pins let you set the pull as well

Reply:
ah okay I see now, thanks for clarifying